.cirrus.yml

```diff
@@ -305,18 +305,6 @@ task:
     - name: deploy_gallery-linux # linux- and macos- only
       depends_on:
         - analyze-linux
-        - framework_tests-widgets-linux
-        - framework_tests-libraries-linux
-        - framework_tests-misc-linux
-        - tool_tests-general-linux
-        - tool_tests-commands-linux
-        - tool_tests-integration-linux
-        - build_tests-linux
-        - hostonly_devicelab_tests-0-linux
-        - hostonly_devicelab_tests-1-linux
-        - hostonly_devicelab_tests-2-linux
-        - hostonly_devicelab_tests-3_last-linux
-        - firebase_test_lab_tests-linux
```

Comment:
Is there any way to make this conditional on what branch we're on?

Maybe even just making two separate tasks, one of which is always skipped where we actually deploy and has no deps, one of which is skipped when we don't deploy and has the deps?

Reply:
Not that I can see from looking in the logs

Reply:
Talked offline - it would be helpful to have a comment here explaining why we think this is safe - it doesn't actually upload on master, by the time it gets to dev it's been throughly tested, adding the dependencies greatly slows down CI

Reply:
Updated with comments, PTAL